src/index.js

```diff
@@ -239,6 +334,11 @@ class Name {
     }
   }
 
+  /**
+   * Get Resolver
+   * @param {string} coinId
```

Comment:
Can you make `coinId` of  `getAddress()`  to be optional? It falls back to `ETH` if it doesn't have an argument so something like https://github.com/ensdomains/ensjs/blob/master/src/__tests__/ens.test.js#L224 will fail

Reply:
The actual fallback logic is here https://github.com/ensdomains/ensjs/blob/7311aebd1125101d8ddaef28b5d3aa4bdc917b57/src/index.js#L244

Reply:
Done. Question actually, do most smart contracts these days prevent sending to 0x000.. ?

Reply:
`Question actually, do most smart contracts these days prevent sending to 0x000.. ?` that is not relevant to this code

Reply:
Ah yes you are right, it is the resolver address, not the smart contract address. I wasn't quite sure if resolvers were smart contracts too or not. Thanks for the clarification